.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -29,8 +30,10 @@ jobs:
           - qemu-tdx-experimental
           - rootfs-image
           - rootfs-initrd
+          - rootfs-initrd-sev
           - shim-v2
           - tdvf
+          - ovmf-sev
```

Comment:
We're trying to keep those alphabetically sorted.  Mind to move it after `nydus` and before `qemu`?